core/server/api/authentication.js

```diff
@@ -167,11 +168,11 @@ authentication = {
      * @param {Object} resetRequest
      * @returns {Promise<Object>} message
      */
-    generateResetToken: function generateResetToken(resetRequest) {
+    generateResetToken: function generateResetToken(object) {
```

Comment:
consistency change. renamed to object because it's the naming convention we use everywhere.